src/inmanta/server/bootloader.py

```diff
@@ -79,10 +78,44 @@ def get_bootstrap_slices(self) -> List[ServerSlice]:
 
     # Extension loading Phase I: from start to setup functions collected
     def _discover_plugin_packages(self) -> List[str]:
+        """ Discover all packages that are defined in the inmanta_ext namespace package. Filter available extensions based on
+            enabled_extensions and disabled_extensions config in the server configuration.
+
+            :return: A list of all subpackages defined in inmanta_ext
+        """
         inmanta_ext = importlib.import_module(EXTENSION_NAMESPACE)
-        return [name for finder, name, ispkg in iter_namespace(inmanta_ext)]
+        available = {name[len(EXTENSION_NAMESPACE) + 1 :]: name for finder, name, ispkg in iter_namespace(inmanta_ext)}
+
+        LOGGER.info("Discoverd extensions: %s", ", ".join(available.keys()))
+
+        extensions = []
+        enabled = [x for x in config.server_enabled_extensions.get() if len(x)]
+
+        if enabled:
+            for ext in enabled:
+                if ext not in available:
+                    raise PluginLoadFailed(
+                        f"Extension {ext} in config option {config.server_enabled_extensions.name} in section "
+                        f"{config.server_enabled_extensions.section} is not available."
+                    )
+
+                extensions.append(available[ext])
+        elif len(available) > 1:
+            # More than core is available
+            LOGGER.info(
+                f"Load extensions by setting configuration option {config.server_enabled_extensions.name} in section "
+                f"{config.server_enabled_extensions.section}. {len(available) - 1} extensions available but none are enabled."
```

Comment:
also put the names of the extensions, so that there is no confusion about what name to use. 
Perhaps even pre-format the config line in the log line.